src/query/query_executor.c

```diff
@@ -6844,6 +6823,22 @@ qexec_close_scan (THREAD_ENTRY * thread_p, ACCESS_SPEC_TYPE * curr_spec)
 	  break;
 	}
     }
+
```

Comment:
This routine can be moved into the prior "switch" caluse with just adding "init btid" logic.

Reply:
No. only if perfmon_is_perf_tracking () is true.